crates/locked-app/src/locked.rs

```diff
@@ -15,7 +15,7 @@ use crate::{
 pub type LockedMap<T> = std::collections::BTreeMap<String, T>;
 
 /// If present and required in `host_requirements`, the host must support
-/// local service chaining (*.spin.internal) or reject the app.
+/// local service chaining (*.spin.alt/.internal) or reject the app.
```

Comment:
nit: Only reading the doc comment it's somewhat unclear whether both are supported.

Reply:
Ugh, that also makes me think that this becomes _another_ compatibility flag because an app that uses `spin.alt` won't work in an environment that supports only `spin.internal`.

Reply:
Given that we're doing this with a major release I think it should be fine to just note how it changed for 3.0.

Reply:
That rather flies in the face of being able to validate that an application runs in _other_ environments though, which was the whole point of adding the host requirements to the lockfile in the first place.

Reply:
I suppose the principled thing would be to add a e.g. `local_service_chaining_alt` host requirement.